stock_dynamic_routing/models/stock_routing_rule.py

```diff
@@ -38,12 +39,19 @@ class StockRoutingRule(models.Model):
         required=True,
         help="Operation type that will be applied on the move.",
     )
+    group_id = fields.Many2one(
+        comodel_name="procurement.group",
+        string="Fixed Procurement Group",
+        help="Operation type that will be applied on the move.",
```

Comment:
?! I guess you mean "Procurement group that will be applied on the move" ?

Reply:
fixed